vspace/src/test/java/edu/asu/diging/vspace/config/ExhibitionLanguageConfigTest.java

```diff
@@ -75,7 +77,12 @@ public void test_init_withNullPropertySource() {
 
     @Test
     public void test_getExhibitionLanguageList_success() {
-        List<Map> languageList = Arrays.asList(Map.of("key1", "value1"), Map.of("key2", "value2"));
+        Map< String, String> map1 = new HashMap();
+        map1.put("key1", "value1") ;
+        Map< String, String> map2 = new HashMap();
+        map1.put("key2", "value2") ;
+        
+        List<Map> languageList = Arrays.asList(map1, map2);
```

Comment:
why was this changed?

Reply:
for flexibility(mutable maps)

Reply:
why? what flexibility is needed here?

Reply:
I did not make any changes in the test files, must be Prachi, baishali, in their previous commits. these are not from my commits

Reply:
should I change it back?

Reply:
Why did you say then that it was for flexibility then? Yes, please change it back.

Reply:
fixed